upgrade of op01no07a reviewed

vzhyu opened the pull request:
https://op-musicology.epfl.ch/projects/harmony-annotations/work_packages/875/

johentsch commented:
Thank you, @vzhyu 

There's a couple of changes that we should discuss further. Also, I haven't touched the cadences but think that in almost all cases the ultima is reached only on the second eighth, don't you think?

vzhyu commented:
Hi Johannes, thanks for your feedback.

I think I understand your point about the cadences; I think I was thinking mostly in the harmonic rhythm of the bass line and didn't consider the melodic line to determine the actual ultima. I changed the phrase endings; is that what you had in mind?

m. 21 I remember being somewhat conflicted about when I annotated it, wondering if that was a phrase at all. I definitely now see your point about the phrase end cutting across the subject. I think the phrase overlap is the best solution here (I moved the phrase ending and the subsequent phrase start to the second eighth note); I'd rather consider that segment its own phrase because it has a clear IAC followed by a HC.

mm. 24-25 I like your fix with the 43, with the 7th of the ii and I chords being implied by the figured bass. I did consider the vi7 ii V7 I model but I felt that the way the chords anticipated the arrival of the ii and I respectively was the more pertinent solution here and also in line with the subsequent ii%43(42) ii%43 V half cadence. Changing to 43 also makes the pattern consistent across the ii, I, and ii%.

m. 27 The reason I placed the V on the quarter note was because I was going by the harmonic rhythm of the figured bass. That said, I now see that the V on the eighth note is probably the more correct option because it follows the melody in the second voice. I'd say I'm in favor of putting it on the eighth note, and maintain this consistency with the other similar cadences.
